src/runtime/math.py

```diff
@@ -87,6 +86,167 @@ def ceil(x: f32) -> i32:
         return r
     return r + 1
 
+# fsum
+# supported data types: i8, i16, i32, i64, f32, f64
+
+@overload
+def fsum(n: i32, arr: i8[:]) -> f64:
```

Comment:
All these functions should follow CPython's API: https://docs.python.org/3/library/math.html#math.fsum

```suggestion
def fsum(iterable: list[i8]) -> f64:
```

We do not support "iterable" yet, so at first just getting it working for lists would be helpful. Later we might support "iterable", so it would start working for other types, such as NumPy arrays as well. In the meantime, one can use NumPy to do such operations to NumPy arrays in LPython.